src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static final int RANDOM_INT = new Random().nextInt(10);
```

Comment:
let's move the logic of random number generation into main method.

Reply:
I did it.